### ISSUE TITLE
Icons are not displayed in users fields+asterisk_logger not starting

### ISSUE THREAD

**Issue by gipsinteractive** [blak3r/yaai#166]
Hi! wonderfull appli... except it does not work at all for me! I' coming back from vTiger...it was really easy to configure.

Maybe your tutos are a little older for the new versions of Sugar C.E?
So, i followed  https://github.com/blak3r/yaai/wiki/User-Manual ...
No troubles for synchronizing manager.conf and Sugar. But impossible to dysplay icon Personal Extension(s), Call notification and Magic dial buttons in user config, despit i did all about your tuto. It is created in auto directly in Studio but i;possible to add them

Also, impossible to start asterisk_logger, here are the fault codes:

root@SERV-GIPS:/etc/init.d# PHP Warning:  fopen(): Filename cannot be empty in /var/www/SuiteCRM/custom/modules/Asterisk/asteriskLogger.php on line 1498
PHP Warning:  fclose() expects parameter 1 to be resource, boolean given in /var/www/SuiteCRM/custom/modules/Asterisk/asteriskLogger.php on line 1505
PHP Notice:  Undefined index: asterisk_log_file in /var/www/SuiteCRM/custom/modules/Asterisk/asteriskLogger.php on line 133
PHP Notice:  Undefined index: asterisk_event_log_file in /var/www/SuiteCRM/custom/modules/Asterisk/asteriskLogger.php on line 134
PHP Warning:  Attempt to modify property of non-object in /var/www/SuiteCRM/include/nusoap/nusoap.php on line 6166
PHP Warning:  Attempt to modify property of non-object in /var/www/SuiteCRM/include/nusoap/nusoap.php on line 6166
PHP Warning:  Attempt to modify property of non-object in /var/www/SuiteCRM/include/nusoap/nusoap.php on line 6166
PHP Warning:  Attempt to modify property of non-object in /var/www/SuiteCRM/include/nusoap/nusoap.php on line 6166

Anybody has an idea?   Thanks

**Comment by blak3r** [comment]
Can you summarize you issue a little more consisley?

The errors aren't really errors. In asteriskLogger. 

What doesn't work?

**Comment by gipsinteractive** [comment]
_Hi_*
*_To be clear_;

1/ it is impossible to get icons: Personal Extension(s), Call 
notification and Magic dial buttons in users panel with the method you 
gave in your doc. The file custom/....List....tpl is already set with 
good parameters and class 'phone'.
In your manual,  you told to create a new panel but your 3 extensions 
are already detected by sugar when you are installing the Callinize 
module. No need to create a special panel. But Anyway, impossible to 
display them in any panel. They are registered in Studio with an *, in 
front of. See attached pics.

2/ _Errors in asterisk_logger_
Impossible to run the script even with USER=www-data or  and Sugar-ROOT= 
/var/www/SuiteCRM

3/As I understood PHP errors are not really problems, but _the script 
does not start at all_. Logs
are just under.

Thanks for your precious help

_Jean-Yves_

**Comment by gipsinteractive** [comment]
_OK...I UNDERSTOOD!:))_*
**
**The problem is coming from translation. Here I got it! For french 
people (at least), they should know some language packs are not taking 
care of translation for special modules as Callinize.**
**Solution: have to log under ''English''ONLY, set the new panel as 
'telephony' for example, add 3 filters in the panel, copy your 3 
extensions inside and finaly relog under french and change the 
tranlastion!! Restart asterisk-logger. And now, your Callinize is 
operating as a charm!!!**
**
*_Thanks!!!!!!!!_

**Comment by blak3r** [comment]
I'm not sure I completely follow.  Could you outline what you had to do in greater detail?
